queries.sql

```diff
@@ -114,4 +114,190 @@ GROUP BY species;
+WHERE m.name = 'Maisy Smith'
+GROUP BY s.name
+ORDER BY num_visits DESC
+LIMIT 1;
+
+
+
+
+
+-- todays task to do 
+
+
+-- Create the vets table
+CREATE TABLE vets (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255),
+    age INTEGER,
+    date_of_graduation DATE
+);
+
+-- Create the species table
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255)
+);
+
+-- Create the specializations join table
+CREATE TABLE specializations (
+    vet_id INTEGER REFERENCES vets(id),
+    species_id INTEGER REFERENCES species(id),
+    PRIMARY KEY (vet_id, species_id)
+);
+
+-- Create the animals table
+CREATE TABLE animals (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255)
+);
+
+-- Create the visits join table
+CREATE TABLE visits (
+    animal_id INTEGER REFERENCES animals(id),
+    vet_id INTEGER REFERENCES vets(id),
+    visit_date DATE,
+    PRIMARY KEY (animal_id, vet_id)
+);
+
+-- Insert data for vets
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES
+    ('William Tatcher', 45, '2000-04-23'),
+    ('Maisy Smith', 26, '2019-01-17'),
+    ('Stephanie Mendez', 64, '1981-05-04'),
+    ('Jack Harkness', 38, '2008-06-08');
+
+-- Insert data for species
+INSERT INTO species (name)
+VALUES
+    ('Pokemon'),
+    ('Digimon');
+
+-- Insert data for specializations
+INSERT INTO specializations (vet_id, species_id)
+VALUES
+    (1, 1), -- William Tatcher - Pokemon
+    (3, 1), -- Stephanie Mendez - Pokemon
+    (3, 2), -- Stephanie Mendez - Digimon
+    (4, 2); -- Jack Harkness - Digimon
+
+-- Insert data for animals
+INSERT INTO animals (name)
+VALUES
+    ('Agumon'),
+    ('Gabumon'),
+    ('Pikachu'),
+    ('Devimon'),
+    ('Charmander'),
+    ('Plantmon'),
+    ('Squirtle'),
+    ('Angemon'),
+    ('Boarmon'),
+    ('Blossom');
+
+-- Insert data for visits
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES
+    (1, 1, '2020-05-24'), -- Agumon -> William Tatcher
+    (1, 3, '2020-07-22'), -- Agumon -> Stephanie Mendez
+    (2, 4, '2021-02-02'), -- Gabumon -> Jack Harkness
+    (3, 2, '2020-01-05'), -- Pikachu -> Maisy Smith
+    (3, 2, '2020-03-08'), -- Pikachu -> Maisy Smith
+    (3, 2, '2020-05-14'), -- Pikachu -> Maisy Smith
+    (4, 3, '2021-05-04'), -- Devimon -> Stephanie Mendez
+    (5, 4, '2021-02-24'), -- Charmander -> Jack Harkness
+    (6, 2, '2019-12-21'), -- Plantmon -> Maisy Smith
+    (6, 1, '2020-08-10'), -- Plantmon -> William Tatcher
+    (6, 2, '2021-04-07'), -- Plantmon -> Maisy Smith
+    (7, 3, '2019-09-29'), -- Squirtle -> Stephanie Mendez
+    (8, 4, '2020-10-03'), -- Angemon -> Jack Harkness
+    (8, 4, '2020-11-04'), -- Angemon -> Jack Harkness
+    (9, 2, '2019-01-24'), -- Boarmon -> Maisy Smith
+    (9, 2, '2019-05-15'), -- Boarmon -> Maisy Smith
+    (9, 2, '2020-02-27'), -- Boarmon -> Maisy Smith
+    (9, 2, '2020-08-03'), -- Boarmon -> Maisy Smith
+    (10, 3, '2020-05-24'), -- Blossom -> Stephanie Mendez
+    (10, 1, '2021-01-11'); -- Blossom -> William Tatcher
+
```

Comment:
- [ ] As I mentioned above :point_up: please use the proper file for proper SQL statements so kindly create one file called `data.sql` and add all `INSERT` statements inside that file and remove all `INSERT` statements from `queries.sql` and `schema.sql` files :slightly_smiling_face: